HOW_WE_WORK.md

```diff
@@ -0,0 +1,7 @@
+# How We Work
+
+## Vendor pull requests
+
+InspiringApps will work on their own fork of the repository. They will file a single pull request at the end of each sprint that includes all work completed in that sprint, as presented at the sprint review. It should be filed within one working day of the end of the sprint, e.g. if the sprint ends on a Thursday, it should be filed by the end of the day on Friday. That pull request will be titled for the sprint (e.g., `Sprint 15`), and the description will summarize the work of the sprint. That will lead with the sprint goal, and then list each completed user story. Each completed user story will include the title, will link back to the story in the backlog, and should include an illustrative screenshot (where applicable).
```

Comment:
We did agree to some different practices, as of this morning:

- We will open pull requests for each feature, as they are completed, after we have completed our own QA/review process
- Pull request names should reflect the feature they encapsulate
- We will open pull requests from a feature branch in the IA fork to the `development` branch in the CSG repo
- CSG will periodically merge their `development` branch into `main` to promote updates in their staging (UAT) environment to production